test/test_fx.py

```diff
@@ -95,6 +95,8 @@ def a_lifted_leaf2(a, b):
 
 wrap('len')
 
+wrap('getattr')
```

Comment:
What if we do this by default inside tracing, e.g. like here: 
https://github.com/pytorch/pytorch/blob/0ef8760bf6b3e8098ef42df60f1e451234151f32/torch/fx/_symbolic_trace.py#L610

I'm primarily curious how this would affect the current semantics of tracing. If it breaks stuff we can skip it

Reply:
I agree.  `getattr` is used extensively and I wasn't quite sure about any possible side effect of patching it.  I saw the builtin function `len` was wrapped so figured we could possibly do the same for `getattr`.

Reply:
Can you run an experiment where you try to patch it here and run all of `test_fx.py` and see if it breaks anything?

Reply:
Will do